doc/manual/citing_aspect.bib

```diff
@@ -891,4 +891,68 @@ @article {BredowM56-2020-2
 	URL = {https://mem.lyellcollection.org/content/early/2021/02/22/M56-2020-2},
 	eprint = {https://mem.lyellcollection.org/content/early/2021/02/22/M56-2020-2.full.pdf},
 	journal = {Geological Society, London, Memoirs}
+
+
+@article{10.1093/gji/ggab051,
+    author = {Saxena, Arushi and Choi, Eunseo and Powell, Christine A and Aslam, Khurram S},
+    title = "{Seismicity in the central and southeastern United States due to upper mantle heterogeneities}",
+    journal = {Geophysical Journal International},
+    volume = {225},
+    number = {3},
+    pages = {1624-1636},
+    year = {2021},
+    month = {03},
+    abstract = "{Sources of stress responsible for earthquakes occurring in the Central and Eastern United States (CEUS) include not only far-field plate boundary forces but also various local contributions. In this study, we model stress fields due to heterogeneities in the upper mantle beneath the CEUS including a high-velocity feature identified as a lithospheric drip in a recent regional P-wave tomography study. We calculate velocity and stress distributions from numerical models for instantaneous 3-D mantle flow. Our models are driven by the heterogeneous density distribution based on a temperature field converted from the tomography study. The temperature field is utilized in a composite rheology, assumed for the numerical models. We compute several geodynamic quantities with our numerical models: dynamic topography, rate of dynamic topography, gravitational potential energy (GPE), differential stress, and Coulomb stress. We find that the GPE, representative of the density anomalies in the lithosphere, is an important factor for understanding the seismicity of the CEUS. When only the upper mantle heterogeneities are included in a model, differential and Coulomb stress for the observed fault geometries in the CEUS seismic zones acts as a good indicator to predict the seismicity distribution. Our modelling results suggest that the upper mantle heterogeneities and structure below the CEUS have stress concentration effects and are likely to promote earthquake generation at preexisting faults in the region’s seismic zones. Our results imply that the mantle flow due to the upper-mantle heterogeneities can cause stress perturbations, which could help explain the intraplate seismicity in this region.}",
+    issn = {0956-540X},
+    doi = {10.1093/gji/ggab051},
+    url = {https://doi.org/10.1093/gji/ggab051},
+    eprint = {https://academic.oup.com/gji/article-pdf/225/3/1624/36583045/ggab051.pdf},
+}
+
+@article{https://doi.org/10.1029/2019JB018726,
+author = {Heyn, Björn H. and Conrad, Clinton P. and Trønnes, Reidar G.},
```

Comment:
Are we allowing special characters now? They are normally escaped in bibtex:
Bj{\"{o}}rn and Tr{\o}nnes

Reply:
thanks Bob